src/Avalonia.Visuals/Animation/Animators/GradientBrushAnimator.cs

```diff
@@ -30,6 +30,7 @@ public class GradientBrushAnimator : Animator<IGradientBrush?>
                     return new ImmutableRadialGradientBrush(
                         InterpolateStops(progress, oldValue.GradientStops, newValue.GradientStops),
                         s_doubleAnimator.Interpolate(progress, oldValue.Opacity, newValue.Opacity),
+                        oldValue.Transform,
```

Comment:
TransformAnimator should be reused or refactored, so we can use Interpolate from there.
Probably should be done in another PR, if it's not straightforward.

Reply:
I think separate PR or maybe @jmacato can give some input here

Reply:
I have added TransformAnimator stub method.

Reply:
Add implementation to TransformAnimator